src/main/java/edu/harvard/iq/dataverse/export/ExportService.java

```diff
@@ -51,9 +58,54 @@ public class ExportService {
 
     private static ExportService service;
     private ServiceLoader<Exporter> loader;
+    private Map<String, Exporter> exporterMap = new HashMap<>();
+
+    private static final Logger logger = Logger.getLogger(ExportService.class.getCanonicalName());
 
     private ExportService() {
-        loader = ServiceLoader.load(Exporter.class);
+        /*
+         * Step 1 - find the EXPORTERS dir and add all jar files there to a class loader
+         */
+        List<URL> jarUrls = new ArrayList<>();
+        Optional<String> exportPathSetting = JvmSettings.EXPORTERS_DIRECTORY.lookupOptional(String.class);
```

Comment:
Just a thought: you might be able to take advantage of auto-conversion via `JvmSettings.EXPORTERS_DIRECTORY.lookupOptional(Path.class)`, as it has a method `Path.of(String)`. (But I would need to try that myself.) You definitely can skip the `String.class`.